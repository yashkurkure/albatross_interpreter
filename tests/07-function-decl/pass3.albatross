fun f1 int () {
  return 1;
}

fun f2 void () {
  return;
}

return 2;
