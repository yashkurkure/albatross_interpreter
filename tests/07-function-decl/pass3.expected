Function declared "f1" returns int
Function declared "f2" returns void
